delfin/drivers/hpe/hpe_msa/ssh_handler.py

```diff
@@ -479,3 +478,319 @@ def handle_xml_to_dict(xml_info, element):
                 for child in children.iter('PROPERTY'):
                     msg[child.get('name')] = child.text
         return msg
+
+    def list_storage_host_initiators(self, storage_id):
+        try:
+            initiator_list = []
+            host_groups_info = self.ssh_pool.do_exec("show initiators")
+            host_groups_json = self.handle_xml_to_json(host_groups_info,
+                                                       "initiator")
+            type_switch = {
+                consts.InitiatorType.ISCSI_INITIATOR_TYPE:
+                    consts.InitiatorType.ISCSI_INITIATOR_DESCRIPTION,
+                consts.InitiatorType.FC_INITIATOR_TYPE:
+                    consts.InitiatorType.FC_INITIATOR_DESCRIPTION,
+            }
+            if host_groups_json:
+                for initiator in host_groups_json:
+                    description = type_switch.get(
+                        initiator.get('host-bus-type-numeric'),
+                        consts.InitiatorType.UNKNOWN_INITIATOR_DESCRIPTION)
+                    initiator_item = {
+                        "name": initiator.get('nickname'),
+                        "type": description,
+                        "alias": initiator.get('durable-id'),
+                        "storage_id": storage_id,
+                        "native_storage_host_initiator_id":
+                            initiator.get('durable-id'),
+                        "wwn": initiator.get('id'),
+                        "status": constants.InitiatorStatus.ONLINE,
+                        "native_storage_host_id": initiator.get('host-id')
+                    }
+                    initiator_list.append(initiator_item)
+            return initiator_list
+        except Exception as e:
+            LOG.error("Failed to get initiators from msa")
+            raise e
+
+    def list_storage_hosts(self, storage_id):
+        try:
+            hosts_info = self.ssh_pool.do_exec('show host-groups')
+            host_list = []
+            hosts = self.handle_xml_to_json(hosts_info, 'host')
+            host_map = {}
```

Comment:
`host_map` is recommended to define to a set type.

Reply:
Change the defined type to set type
![image](https://user-images.githubusercontent.com/88076879/155679874-7158925f-f6fa-48dd-b6a8-735e74da8694.png)